doc/source/train/examples/lightning/lightning_mnist_example.ipynb

```diff
@@ -232,7 +225,7 @@
     "    # )\n",
     "    config_builder.trainer(\n",
     "        max_epochs=10,\n",
-    "        accelerator=\"cpu\",\n",
+    "        accelerator=\"gpu\" if use_gpu else \"cpu\",\n",
```

Comment:
Should the user even need to set this? Can this be inferred and set from the `ScalingConfig`, and if not can we just raise a warning?